src/beacon-api/cronjobs/update_devices.py

```
@@ -164,20 +168,60 @@ def parse_device_from_api(self, api_device: Dict[str, Any]) -> Dict[str, Any]:
             try:
                 last_updated = datetime.fromisoformat(online_status['lastUpdate'].replace('Z', '+00:00'))
             except (ValueError, AttributeError):
-                logger.warning(f"Could not parse lastUpdate for device {api_device.get('_id')}")
+                # Fallback to lastRawData if available
+                if api_device.get('lastRawData'):
+                    try:
+                         last_updated = datetime.fromisoformat(api_device['lastRawData'].replace('Z', '+00:00'))
+                    except:
```

Ruff (0.14.13)

Do not use bare `except`

(E722)